lib/statsd.js

```diff
@@ -48,6 +51,8 @@ const Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
   this.cacheDnsTtl = options.cacheDnsTtl || CACHE_DNS_TTL_DEFAULT;
   this.host = options.host || process.env.DD_AGENT_HOST;
   this.port = options.port || parseInt(process.env.DD_DOGSTATSD_PORT, 10) || 8125;
+  this.path = options.path;
```

Comment:
- I've added **four** public fields to `Client` (`path`, `stream`, `udsGracefulRestartRateLimit` and `isChild`), this may not be a desirable addition to the "exported" interface
- I haven't updated the TypeScript definition(s) to reflect new exports / fields

Reply:
That's fine, and you shouldn't add these to TypeScript definitions, others ones aren't mentioned in there like this (I don't think)